apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -179,7 +181,7 @@ def is_vertex_cover(G: nx.DiGraph, vertex_cover: Iterable[str]) -> bool:
 
 
 def lcs_partial_credit(
-    submission: list[str],
+    submission: Sequence[str | None],
```

Comment:
@eliotwrobson The test suite has submissions with `None` in them. Do those even need to be handled? I didn't test this code.

Reply:
@SethPoulsen is the one to ask about tests for this element

Reply:
Yes, we need to handle `None`. 

This is for the same reason as my comment on the other PR, if a student submits an answer with an incorrect block in it, it is represented as `None` in the submission list.

Reply:
Resolved in [10c9d37](https://github.com/PrairieLearn/PrairieLearn/pull/11249/commits/10c9d37f2093c05684b81aeb37e5f3e32073056f).